praw/const.py

```diff
@@ -171,7 +171,11 @@
     'wiki_page_revisions':    'r/{subreddit}/wiki/revisions/{page}',
     'wiki_page_settings':     'r/{subreddit}/wiki/settings/{page}',
     'wiki_pages':             'r/{subreddit}/wiki/pages/',
-    'wiki_revisions':         'r/{subreddit}/wiki/revisions/'}
+    'wiki_revisions':         'r/{subreddit}/wiki/revisions/',
```

Comment:
Can you insert these in a sorted order please?

Reply:
Done! Wanted to make the changes most visible, but this makes sense.